src/include/duckdb/logging/httplib_logger.hpp

```diff
@@ -9,29 +9,37 @@
 #pragma once
 
 #include "duckdb/common/fstream.hpp"
-#include "duckdb/common/mutex.hpp"
-#include "duckdb/common/printer.hpp"
 #include "duckdb/main/client_context.hpp"
+#include "duckdb/logging/log_type.hpp"
 
 #include <functional>
 
-// TODO unify with new logger infra in duckdb/logging/logging.hpp
-
 namespace duckdb {
 
+//! This class is used to hook up the httplib (./third_party/httplib) logger to the duckdb logger.
 //! This has to be templated because we have two namespaces:
 //! 1. duckdb_httplib
 //! 2. duckdb_httplib_openssl
 //! These have essentially the same code, but we cannot convert between them
 //! We get around that by templating everything, which requires implementing everything in the header
-class HTTPLogger {
+class HTTPLibLogger {
```

Comment:
It might be nice to add a wrapper to the httplib client that has this logger as well